tests/functional/s3cmd/tests.js

```diff
@@ -691,22 +691,14 @@ describe('s3cmd put, get and delete object with spaces ' +
 });
 
 describe('s3cmd info', () => {
-// test that CORS and POLICY are returned as 'none'
+// test that POLICY are returned as 'none'
     it('should find that policy has a value of none', done => {
         checkRawOutput(['info', `s3://${bucket}`], 'policy', 'none',
         foundIt => {
             assert(foundIt);
             done();
         });
     });
-
-
-    it('should find that cors has a value of none', done => {
```

Comment:
might as well test that our implementation of put cors is compatible with s3cmd.  can skip until get is implemented if needed.